### PR TITLE
Align binding (script message and channel) protocol with the spec

### PR DESCRIPTION
**Pull request by thiagowfx**
References:

- `s/OwnershipModel/ResultOwnership/` to align with the spec
- `script.Channel`: https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/pull/361.html#type-script-Channel
- `script.Message`: https://pr-preview.s3.amazonaws.com/w3c/webdriver-bidi/pull/361.html#event-script-message

Bug: #294
Reference: https://github.com/GoogleChromeLabs/chromium-bidi/pull/319#issuecomment-1453236068

### CODE REVIEW

**Comment by thiagowfx** [comment]
Forked off https://github.com/GoogleChromeLabs/chromium-bidi/pull/319, in order to implement `script.Message` first.

**Review by sadym-chromium — APPROVED** [review]

**Comment by thiagowfx** [comment]
@sadym-chromium

1. Should I introduce a new `Script.Event` namespace and put the added event therein?
2. Is `ScriptBindingCalledEvent` still OK, or should we choose another name?
3. Do I need to define _all_ remote schemas in `RemoteValueSchema`?
4. Where do I use `MessageParametersSchema`?
5. Should `Channel` be within `Script` or within `CommonDataTypes`?

File: src/protocol/protocol.ts [hunk 1]
```diff
@@ -656,7 +682,8 @@ export namespace BrowsingContext {
     | LoadEvent
     | DomContentLoadedEvent
     | ContextCreatedEvent
-    | ContextDestroyedEvent;
+    | ContextDestroyedEvent
+    | ScriptBindingCalledEvent;
```
Comment (sadym-chromium) [inline src/protocol/protocol.ts:686]: It should be in `Script` namespace

Comment (thiagowfx) [reply src/protocol/protocol.ts:686]: `MessageEvent` instead of Binding as per the other comment

**Comment by thiagowfx** [comment]
Addressed all your comments.

**Comment by thiagowfx** [comment]
@sadym-chromium If I add `script.Channel` to `ArgumentValue`, the tests fail.

Any clue what could be wrong?

Either way I am going ahead with this PR without this specific part, to unblock the other PR.

**Pull request merged**